R/PipeOpPredictionUnion.R

```diff
@@ -0,0 +1,136 @@
+#' @title PipeOpPredictionUnion
```

Comment:
1) please do separate pipeops in separate PRs
2) shouldn't the ensembling pipeops handle this case?

Reply:
dropped the union stuff of predictions from this PR for now